src/Platform/AcquiaCloudPlatform.php

```diff
@@ -182,14 +192,31 @@ public static function getEnvironmentQuestion(Config $config, AcquiaCloudClientF
    * {@inheritdoc}
    */
   public function execute(Command $command, InputInterface $input, OutputInterface $output) : int {
+    if ($input->getOption('group') && $input->getOption('uri')) {
+      $helper = $command->getHelper('question');
+      $question = new ConfirmationQuestion('You have provided both the options, group as well as uri. We will ignore the uri option. Do you want to proceed (y/n)?', TRUE);
+      if (!$helper->ask($input, $output, $question)) {
+        return 1;
+      }
+    }
+
     $environments = new Environments($this->getAceClient());
```

Comment:
I would still start with this part. If there is no sites at all in the platform, then do not display the duplicated options message. Tell them first that something is wrong with the platform.

Reply:
Changed. Thanks.